src/test/java/marquez/db/ColumnsTest.java

```diff
@@ -27,16 +26,25 @@
 import java.util.List;
 import java.util.UUID;
 import marquez.UnitTests;
+import org.junit.Rule;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.mockito.quality.Strictness;
 
 @Category(UnitTests.class)
 public class ColumnsTest {
+  @Rule public MockitoRule rule = MockitoJUnit.rule().strictness(Strictness.WARN);
```

Comment:
Should we be using `Strictness.STRICT_STUBS` instead?

Reply:
@wslulciuc 
In each of the test cases, different values has been used for column argument in ResultSet mock method, so had to choose Strictness.WARN. If we had to use Strictness.STRICT_STUBS, then each of the test cases has to be updated to use Mockito.anyString() as argument in ResultSet mock methods. Please let me know, how do we want to handle this.

Note: Strictness.WARN has also been used in below test classes, due to above reason
NamespaceRowMapperTest
JobServiceTest

Reply:
Great, thanks for clarifying @ravikamaraj! Looks like `Strictness.WARN` has been the default behavior since Mockito [2.x](https://static.javadoc.io/org.mockito/mockito-core/2.27.0/org/mockito/quality/Strictness.html) allowing us to define the rule simply as:

```java
@Rule public MockitoRule rule = MockitoJUnit.rule()
```

Reply:
yes, that's correct, I would update the code shortly

Reply:
done, can you review